src/scheduler/workers/unpublish/dayBefore/addUnpublishReminderAudit.ts

```diff
@@ -0,0 +1,18 @@
+import { prisma } from "server/models/db/prisma-client";
+import { AuditCreateInput, ListEventJsonData } from "server/models/types";
+import { AuditEvent } from "@prisma/client";
+
+export function recordListItemEvent(
+  eventData: ListEventJsonData,
+  auditEvent: AuditEvent,
+  type?: "user" | "list" | "listItem"
+) {
+  type = type ?? "listItem";
```

Comment:
If you feel like bing fancy
```suggestion
  type??= "listItem";
```

But of course this is an optional change.